easybuild/easyconfigs/g/GLib/GLib-2.49.5-GCCcore-5.4.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'GLib'
+version = '2.49.5'
+
+homepage = 'http://www.gtk.org/'
+description = """GLib is one of the base libraries of the GTK+ project"""
+
+toolchain = {'name': 'GCCcore', 'version': '5.4.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+
+dependencies = [
+    ('libffi', '3.2.1'),
+    ('gettext', '0.19.8'),
+    ('libxml2', '2.9.4'),
+    ('PCRE', '8.39'),
+]
+
+#GLib compiles correctly even without this dependency
+#builddependencies = [('Python', '2.7.12')]
```

Comment:
Ah, yes, that compilates things a bit...

Although, a bare Python at `GCCcore` should be fine, no?